actors/miner/src/types.rs

```diff
@@ -273,7 +277,34 @@ pub struct SectorPreCommitInfo {
     pub replace_sector_number: SectorNumber,
 }
 
+#[derive(Debug, Default, PartialEq, Clone, Serialize_tuple, Deserialize_tuple)]
+pub struct SectorPreCommitInfo2 {
+    pub seal_proof: RegisteredSealProof,
+    pub sector_number: SectorNumber,
+    /// CommR
+    pub sealed_cid: Cid,
+    pub seal_rand_epoch: ChainEpoch,
+    pub deal_ids: Vec<DealID>,
+    pub expiration: ChainEpoch,
+    /// CommD
+    pub unsealed_cid: Option<Cid>,
+}
+
 /// Information stored on-chain for a pre-committed sector.
+#[derive(Debug, PartialEq, Clone, Serialize_tuple, Deserialize_tuple)]
+pub struct SectorPreCommitOnChainInfo2 {
```

Comment:
We should be able to change the existing type rather than create a new one. I understand the old one should be un-used in the new code. We'll need both only in migration code.

I'm guessing you ran into the unfortunate re-use of `SectorPreCommitInfo` as both state type and the exported parameter type. We should fix this by redefining `PreCommitSectorParams` to be a copy of the state type rather than identified with it.

Reply:
I left the old state affecting types as they are because I don't know what is the migration flow for FVM.

Reply:
If we are sure that they are not needed, I can remove them and run a refactor.